app.rambox.ramboxce.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: app.rambox.ramboxce
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/linux-gnu/GL
+    version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
```

Comment:
`--talk-name=org.kde.StatusNotifierWatcher` is needed, otherwise the X11 tray icon is used.  However, the original icon name needs to be available too.